File: src/dart/vaden/lib/models/cache_response.dart
class CacheResponse {
  final String key;
  final String value;
  final bool cached;
  final int ttl;

  const CacheResponse({
    required this.key,
    required this.value,
    required this.cached,
    required this.ttl,
  });

  Map<String, dynamic> toJson() {
    return {
      'key': key,
      'value': value,
      'cached': cached,
      'ttl': ttl,
    };
  }
}

File: src/csharp/FastEndpoints/Models/CacheResponse.cs
namespace BenchmarkFastEndpoints.Models;

public class CacheResponse
{
    public string Key { get; set; } = string.Empty;
    public string Value { get; set; } = string.Empty;
    public bool Cached { get; set; }
    public int Ttl { get; set; } = 300;
    public DateTime Timestamp { get; set; } = DateTime.UtcNow;
}
